### PR TITLE
LPS-187212 [PRM] Add the "Approved" and "Paid" info for the MDF Request and Claim List

### PR DESCRIPTION
**Pull request by JoMaAlves**
Added modifications to this PR: https://github.com/pt-liferay-solutions/liferay-portal/pull/4333

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro [hunk 1]
```diff
@@ -286,9 +286,9 @@ definition {
 				locator1 = "PRMMDFClaim#SUMMARY_PANEL");
 		}
 
-		if (isSet(paymentReceived)) {
+		if (isSet(claimPaid)) {
 			AssertElementPresent(
-				key_itemValue = ${paymentReceived},
+				key_itemValue = ${claimPaid},
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro:291]: Remove those changes.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/utils/getMDFBudgetInfos.ts [hunk 2]
```diff
@@ -16,7 +16,8 @@ import getIntlNumberFormat from '../../../common/utils/getIntlNumberFormat';
 export default function getMDFBudgetInfos(
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/utils/getMDFBudgetInfos.ts:16]:
```suggestion
const APPROVED_STATUS = 'approved';

export default function getMDFBudgetInfos(
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/utils/getMDFBudgetInfos.ts [hunk 3]
```diff
@@ -26,6 +27,10 @@ export default function getMDFBudgetInfos(
 			[MDFColumnKey.REQUESTED]: getIntlNumberFormat(currency).format(
 				totalRequested
 			),
+			[MDFColumnKey.APPROVED]:
+				requestStatus === 'approved'
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestList/utils/getMDFBudgetInfos.ts:31]:
```suggestion
				requestStatus === APPROVED_STATUS
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by JoMaAlves** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**